### ISSUE TITLE
[Linux] missing "sensitive" accessibility state

### ISSUE THREAD

**Issue by jpnurmi** [flutter/flutter#101500]
Orca Screen Reader considers Flutter widgets "greyed" because the `ATK_STATE_SENSITIVE` flag is not set.

## Steps to Reproduce

1. Turn on the screen reader (Settings -> Accessibility -> Screen Reader) 
2. Execute `flutter run` on the code sample 
3. Press _Tab_ to navigate around 

**Expected results:** 

All three widgets are enabled so they should not be read as "greyed".

**Actual results:** 

Enabled checkboxes, radio buttons, and switches are read as "greyed".

<details>
<summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  final checked = ValueNotifier<bool>(true);

  runApp(
    MaterialApp(
      home: Scaffold(
        body: ValueListenableBuilder<bool>(
          valueListenable: checked,
          builder: (context, value, _) {
            return Column(
              children: [
                Checkbox(
                  value: value,
                  onChanged: (v) => checked.value = v!,
                ),
                Radio<bool>(
                  value: value,
                  groupValue: true,
                  onChanged: (v) => checked.value = v!,
                ),
                Switch(
                  value: value,
                  onChanged: (v) => checked.value = v,
                )
              ],
            );
          },
        ),
      ),
    ),
  );
}
```

</details>

<details>
  <summary>Logs</summary>

```
[✓] Flutter (Channel master, 2.13.0-0.0.pre.393, on Ubuntu Jammy Jellyfish (development branch) 5.15.0-25-generic, locale en_US.UTF-8)
    • Flutter version 2.13.0-0.0.pre.393 at /home/jpnurmi/Flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 94fefaa49d (5 hours ago), 2022-04-06 17:17:34 +0530
    • Engine revision 45bedbe841
    • Dart version 2.17.0 (build 2.17.0-266.0.dev)
    • DevTools version 2.12.1

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /home/jpnurmi/Android/Sdk
    • Platform android-31, build-tools 31.0.0
    • ANDROID_SDK_ROOT = /home/jpnurmi/Android/Sdk
    • Java binary at: /home/jpnurmi/Android/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Linux toolchain - develop for Linux desktop
    • Ubuntu clang version 14.0.0-1ubuntu1
    • cmake version 3.22.1
    • ninja version 1.10.1
    • pkg-config version 0.29.2

[✓] Android Studio (version 2020.3)
    • Android Studio at /home/jpnurmi/Android/android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code
    • VS Code at /snap/code/current
    • Flutter extension version 3.38.1

[✓] Connected device (1 available)
    • Linux (desktop) • linux • linux-x64 • Ubuntu Jammy Jellyfish (development branch) 5.15.0-25-generic

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

</details>

https://user-images.githubusercontent.com/140617/162153311-61abe9f4-fa98-41a5-a227-04428df4b3ef.mp4

**Comment by jpnurmi** [comment]
Orca logs when interacting with a Flutter Checkbox:
```
11:34:21.263037 - OBJECT EVENT: object:state-changed:focused (1, 0, 0)
                                app.name='flutter_application_11' name='CB' role='check box'
                                description=''
                                states='checked enabled focusable focused showing visible checkable'
                                relations=''
                                interfaces='Accessible Action Collection Component Text'
                                attributes=''
                                path=[-1, 0, 0, 0, 0, 0, 0, 0]
```
When interacting with a GtkCheckButton:
```
11:40:29.912171 - OBJECT EVENT: object:state-changed:focused (1, 0, 0)
                                app.name='glade-previewer' name='checkbutton' role='check box'
                                description=''
                                states='checked enabled focusable focused sensitive showing visible'
                                relations=''
                                interfaces='Accessible Action Collection Component Image'
                                attributes='toolkit:gtk'
                                path=[-1, 0, 0, 0]
```

**Comment by maheshmnj** [comment]
Hi @jpnurmi, Thanks for filing the issue. I am not able to reproduce the issue  completely

*Unmute the video for audiovisual output*

### output from stable

*screen reader reads "Frame greyed"  regardless of whether the widget is enabled or not*

https://user-images.githubusercontent.com/31410839/162206570-e31f0670-2a55-4c2d-9e7c-15055581b6f6.mp4

### output from master

*Screen reader readers correctly when the mouse is hovered, However focusing via Tab key only reads the Tab button*

https://user-images.githubusercontent.com/31410839/162213449-9424c2e4-127b-4451-9ea0-5921202782ee.mp4

Labeling this issue for further investigation.

<details>
<summary>flutter doctor -v (Linux)</summary>

```
[✓] Flutter (Channel stable, 2.10.4, on Ubuntu 20.04.4 LTS 5.13.0-30-generic, locale en_GB.UTF-8)
    • Flutter version 2.10.4 at /home/mahesh/Development/stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision c860cba910 (12 days ago), 2022-03-25 00:23:12 -0500
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /home/mahesh/Android/Sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[✓] Android Studio (version 2020.3)
    • Android Studio at /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio
    • Flutter plugin version 63.2.1
    • Dart plugin version 203.8452
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.64.2)
    • VS Code at /usr/share/code
    • Flutter extension version 3.38.1

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Ubuntu 20.04.4 LTS 5.13.0-30-generic
    • Chrome (web)    • chrome • web-javascript • Google Chrome 98.0.4758.102

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
```
[✓] Flutter (Channel master, 2.13.0-0.0.pre.392, on Ubuntu 20.04.4 LTS 5.13.0-30-generic, locale en_GB.UTF-8)
    • Flutter version 2.13.0-0.0.pre.392 at /home/mahesh/Development/master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 120b3deb18 (20 minutes ago), 2022-04-06 10:59:37 +0200
    • Engine revision 45bedbe841
    • Dart version 2.17.0 (build 2.17.0-266.0.dev)
    • DevTools version 2.12.1

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /home/mahesh/Android/Sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[✓] Android Studio (version 2020.3)
    • Android Studio at /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio
    • Flutter plugin version 63.2.1
    • Dart plugin version 203.8452
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.64.2)
    • VS Code at /usr/share/code
    • Flutter extension version 3.38.1

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Ubuntu 20.04.4 LTS 5.13.0-30-generic
    • Chrome (web)    • chrome • web-javascript • Google Chrome 98.0.4758.102

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by gspencergoog** [comment]
cc @cbracken
